yaml/native/native_functions.yaml

```diff
@@ -4166,6 +4166,18 @@
   dispatch:
     XPU: glu_backward_xpu
 
+- func: glu_jvp(Tensor glu, Tensor x, Tensor dx, int dim) -> Tensor
+  python_module: nn
+  dispatch:
+    XPU: glu_jvp
+  autogen: glu_jvp.out
+
+- func: glu_backward_jvp(Tensor grad_x, Tensor grad_glu, Tensor x, Tensor dgrad_glu, Tensor dx, int dim) -> Tensor
+  python_module: nn
+  dispatch:
+    XPU: glu_backward_jvp
```

Comment:
No backward kernel?

Reply:
Pytorch implemented this by concatenating some aten ops at [link](https://github.com/pytorch/pytorch/blob/v2.5.1/aten/src/ATen/native/GatedLinearUnit.cpp#L111), cuda doesn't have **glu_backward_jvp** kernel yet.